components/kyma-environment-broker/internal/storage/driver/postsql/instance.go

```diff
@@ -69,3 +69,8 @@ func (s *Instance) Update(instance internal.Instance) error {
 		return true, nil
 	})
 }
+
+func (s *Instance) Delete(instanceID string) error {
+	sess := s.NewWriteSession()
+	return sess.DeleteInstance(instanceID)
```

Comment:
question: every other interaction with postsql database includes retries - is it okay to not use `wait.Poll` in case of deleting?

Reply:
In general, I don't like idea of having retries on this level, but I could add it here like for other operations. The problem is currently the deprovisioning call is synchronous, so it should not take too much time.